### ISSUE TITLE
nothing committed (yet) on El Capitan

### ISSUE THREAD

**Issue by jbrooksuk** [SublimeGit/SublimeGit#208]
I've updated to El Capitan and now the Git Status buffer is showing:

```
Local:    (no branch) ~/Code/CachetHQ/Cachet
Head:    nothing committed (yet)

Nothing to commit (working directory clean)
```

This was working before the update.

**Comment by Axik** [comment]
+1 here

**Comment by jbrooksuk** [comment]
I upgraded Homebrew and then this sort of solved itself.

**Comment by Axik** [comment]
Helped me too. But what the root of issue?

**Comment by jbrooksuk** [comment]
Probably something El Capitan related.

**Comment by domibarton** [comment]
Probably a new XCode version or you've to accept the terms again? Did you try to run `git` on the command line?

**Comment by Newt0n** [comment]
+1

`brew update` finished to the newest, `git version` 2.6.1, still not work, anybody have solution?

**Comment by maxheld83** [comment]
+1.

I got `which git` `/usr/local/bin/git` at version `git version 2.6.2` and SublimeGit used to work just until the El Cap update.

I ran `brew upgrade`, `brew update`, `brew doctor`, `brew prune`, and some `brew link`s, all to no avail.

**Forcing the git path by appending `"git_force_path": "/usr/local/bin"` to `SublimeGit.sublime-settings` did the trick.**

Odd.

Would be nice if this kind of thing could be sorted out *before* an OS X update ...
